src/main/kotlin/com/sainsburys/k2zpl/command/barcode/BarCode39.kt

```diff
@@ -34,32 +38,29 @@ internal data class BarCode(
  * @param data data encoded in the barcode
  * @param x horizontal position
  * @param y vertical position
- * @param barcodeType Barcode type
  * @param orientation The orientation of the barcode.
- * @param checkDigit Whether to include a check digit.
+ * @param checkDigit Mod-43 check digit
  * @param height The height of the barcode.
- * @param lineThickness The line thickness of the barcode.
- * @param lineAbove Whether to include a line above the barcode.
+ * @param interpretationLine print interpretation line
+ * @param lineAbove print interpretation line above code
  */
```

Comment:
Similarly to my other comment, I believe common Javadoc (and therefore KDoc) practice for `@param` tags is to not start with capitals, have no full stops and separate "sentences" with semi-colons.